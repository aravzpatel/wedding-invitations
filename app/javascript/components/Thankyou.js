import React from "react";
import thankyouArtwork from "../../assets/images/Thankyou.svg";
import Image from "react-bootstrap/Image";
class Thankyou extends React.Component {
  render () {
    return (
      <Image 
        src={thankyouArtwork} 
        alt="Image" 
        fluid
      />
    );
  }
}

export default Thankyou;
