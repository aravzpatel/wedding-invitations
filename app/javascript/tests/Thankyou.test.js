import React from "react";
import Thankyou from "../components/Thankyou";
import renderer from "react-test-renderer";

it("Thankyou component renders without an rsvp button", () => {
  const component = renderer.create(
    <Thankyou/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});