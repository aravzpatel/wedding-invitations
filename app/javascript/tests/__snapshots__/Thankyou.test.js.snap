// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thankyou component renders without an rsvp button 1`] = `
<img
  alt="Image"
  className="img-fluid"
  src="ArtworkMock"
/>
`;
